app/models/docker_builder_service.rb

```diff
@@ -61,7 +61,6 @@ def before_docker_build(tmp_dir)
     Samson::Hooks.fire(:before_docker_build, tmp_dir, @build, @output)
     execute_build_command(tmp_dir, @build.project.build_command)
   end
-  add_method_tracer :before_docker_build
```

Comment:
the calls close to the method were nice to not forget to rename them / to be aware that they are instrumented

Reply:
right!!, but if there are multiple methods calls, defining into single trace method call, reduces the execution timing.